src/resources/js/tickets.js

```diff
@@ -371,9 +379,29 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 				params,
 				function( response ) {
 					$tribe_tickets.trigger( 'saved-ticket.tribe', response );
+
+					if ( response.success ) {
+						console.log('SUCCESS');
+						var responseData = JSON.parse( response.data.data );
```

Comment:
Shouldn't we parse this by passing on the params that this will always return a JSON?